perun-core/src/main/java/cz/metacentrum/perun/core/entry/AttributesManagerEntry.java

```diff
@@ -4512,6 +4512,19 @@ public void setAttributeRights(PerunSession sess, List<AttributeRights> rights)
 		getAttributesManagerBl().setAttributeRights(sess, rights);
 	}
 
+	@Override
+	public void setAttributePolicyCollections(PerunSession sess, List<AttributePolicyCollection> policies, int attributeId) throws PrivilegeException, AttributeNotExistsException {
+		Utils.checkPerunSession(sess);
+		getAttributeDefinitionById(sess, attributeId);
+
```

Comment:
I think we should check here whether roles of given attribute policy collections exist. Similarly like in `setAttributeRights`.